CONTRIBUTING.md

```diff
@@ -37,19 +37,19 @@ Developer's Certificate of Origin 1.1
 By making a contribution to this project, I certify that:
 
 (a) The contribution was created in whole or in part by me and I
-    have the right to submit it under the open source license
+    have the right to submit it under the open-source license
     indicated in the file; or
 
 (b) The contribution is based upon previous work that, to the best
-    of my knowledge, is covered under an appropriate open source
+    of my knowledge, is covered under an appropriate open-source
     license and I have the right under that license to submit that
     work with modifications, whether created in whole or in part
-    by me, under the same open source license (unless I am
+    by me, under the same open-source license (unless I am
     permitted to submit under a different license), as indicated
     in the file; or
 
 (c) The contribution was provided directly to me by some other
-    person who certified (a), (b) or (c) and I have not modified
+    the person who certified (a), (b) or (c) and I have not modified
```

Comment:
This needs to be corrected. And please, as @rohanKanojia mentioned, don't touch the Developer's Certificate of Origin part as we do not own this; we just replicate it here. If you want to update this, please get in touch with the Linux Foundation, when they change it, we will take it over here. Consider it like a law text, you can't change it even with errors.

Reply:
> This needs to be corrected. And please, as @rohanKanojia mentioned, don't touch the Developer's Certificate of Origin part as we do not own this; we just replicate it here. If you want to update this, please get in touch with the Linux Foundation, when they change it, we will take it over here. Consider it like a law text, you can't change it even with errors.

I deleted that commit that I changed that Developer's Certificate. Now I just made changes only in ```into.md``` and ```CONTRIBUTING.md```